assets/src/components/mapPage.tsx

```diff
@@ -150,22 +166,28 @@ const MapPage = (): ReactElement<HTMLDivElement> => {
           reactLeafletRef={leafletMap}
           vehicles={onlyVehicles}
           onPrimaryVehicleSelect={selectVehicle}
-          shapes={showVehicleCard ? selectedVehicleShapes : undefined}
+          shapes={vpcEnabled ? selectedVehicleShapes : undefined}
           allowStreetView={true}
           stopCardDirection={liveVehicle?.directionId}
           includeStopCard={true}
           stations={stations}
           selectedVehicleId={selectedVehicleId ?? undefined}
         >
-          {vpcEnabled ? (
-            <VehiclePropertiesCard
-              vehicle={liveVehicle}
-              onClose={() => {
-                dispatch(setSelectedVehicle(null))
-                setShowVehicleCard(false)
-              }}
-            />
-          ) : undefined}
+          <>
+            {selectedVehicle && isVehicle(selectedVehicle) && <></>}
```

Comment:
**question:** is this fragment a placeholder for something? Right now I don't think it actually changes anything about the DOM if `selectedVehicle` is present and also a `Vehicle`.

Reply:
yeah that's where the autocenterer is going